sigh/lib/assets/resign.sh

```diff
@@ -447,25 +470,18 @@ function resign {
     log "Current bundle identifier is: '$CURRENT_BUNDLE_IDENTIFIER'"
     log "New bundle identifier will be: '$BUNDLE_IDENTIFIER'"
 
-    # Update the CFBundleDisplayName property in the Info.plist if a new name has been provided
-    if [ "${DISPLAY_NAME}" != "" ];
-    then
-        if [ "${DISPLAY_NAME}" != "${CURRENT_NAME}" ];
-        then
-            log "Changing display name from '$CURRENT_NAME' to '$DISPLAY_NAME'"
-            PlistBuddy -c "Set :CFBundleDisplayName $DISPLAY_NAME" "$APP_PATH/Info.plist"
-        fi
-    fi
+    change_plist_value "$INFO_PLIST" "CFBundleDisplayName" "$DISPLAY_NAME"
 
     # Replace the embedded mobile provisioning profile
     log "Validating the new provisioning profile: $NEW_PROVISION"
-    security cms -D -i "$NEW_PROVISION" > "$TEMP_DIR/profile.plist"
+    PROFILE_PLIST="$TEMP_DIR/profile.plist"
```

Comment:
no `local` as with the other vars? (I have no idea of bash...)

Reply:
I would stick with `local` as this variable is not used outside of the function. At my company, we keep local variables lowercase to make it more obvious.